helpers/__init__.py

```diff
@@ -0,0 +1,35 @@
+import os
+import sys
+import platform
+
+# Various helpers for the build scripts
+
+def get_lib_dir():
+    if sys.platform.startswith("win"):
+        if platform.architecture()[0] == '32bit':
+            # 32 bit
+            return ['prebuilt-x86/lib']
+        else:
+            # 64 bit
+            return ['prebuilt-x64/lib']
+    return ['/usr/lib']
+
+
+def get_inc_dir():
+    if sys.platform.startswith("win"):
+        if platform.architecture()[0] == '32bit':
+            # 32 bit
+            return ['prebuilt-x86/include', 'prebuilt-x86/include/SDL']
+        else:
+            return ['prebuilt-x64/include', 'prebuilt-x64/include/SDL']
+    return ['/usr/include', '/usr/include/SDL', '/usr/local/include/SDL']
+
```

Comment:
I think we might be missing a newline between these two functions. Should we perhaps add docstrings to these?